File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.py
def solution(my_string, overwrite_string, s):
    list_1 = list(my_string)
    list_2 = list(overwrite_string)
    j = 0
    
    for i in range(s, s + len(list_2)):
        list_1[i] = list_2[j]
        j += 1
    answer = ''.join(list_1)
        
    return answer

File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.js
function solution(my_string, overwrite_string, s) {
    let list1 = my_string.split('')
    let list2 = overwrite_string.split('')
    let j = 0
    
    
    for(let i = s; i < s + list2.length; i++){
        list1[i] = list2[j]; // 배열의 특정 인덱스에 값을 할당하는 문법
        j++;
    }
    
    return list1.join('');
}

